node/src/chain_spec.rs

```diff
@@ -29,14 +40,44 @@ pub type ChainSpec = sc_service::GenericChainSpec<TuxedoGenesisConfig>;
 // 	(get_from_seed::<AuraId>(s), get_from_seed::<GrandpaId>(s))
 // }
```

Comment:
@JoshOrndorff can this commented part be removed?

Reply:
https://github.com/Off-Narrative-Labs/Tuxedo/blob/52b0b49e0930e10e71b3cdddf0f519cb854babe1/node/src/chain_spec.rs#L21-L41

Reply:
This one as well:
https://github.com/Off-Narrative-Labs/Tuxedo/blob/52b0b49e0930e10e71b3cdddf0f519cb854babe1/node/src/chain_spec.rs#L15-L16

Reply:
The telemetry address should stay. It is [copied directly](https://github.com/paritytech/polkadot-sdk/blob/6b7be115fd523057d01cea46e9680a62a0b08d97/substrate/bin/node-template/node/src/chain_spec.rs#L9) from the upstream node template (commented and all).

The reason they have it there is because production chains will want it enabled and it is nice to have it readily available. The reason it is commented is because Parity doesn't want every learner who spins up a node template spamming their telemetry server.

Basically, one of my goals is to keep the diff between our node template's client side and the upstream node template's client side as small as practical.

Reply:
Got it. Let me know about the other part then.

Reply:
For the main question about get_from_seed and stuff. I basically just haven't fully understood this stuff. I don't know if we will be useful when we start having proper authority selection in the runtime, etc. I don't think we should remove it in this PR at least. I'll make an issue describing some ideas I have in mind about authority selection.

Reply:
Let me know if we can merge this, so I can move on with #132 as well